provider/pkg/provider/webhook.go

```diff
@@ -424,15 +496,18 @@ func (wh *PulumiServiceWebhookResource) getWebhook(id string) (*pulumiapi.Webhoo
 		return nil, err
 	}
 	webhook, err := wh.client.GetWebhook(context.Background(),
-		hookID.organizationName, hookID.projectName, hookID.stackName, hookID.webhookName)
+		hookID.organizationName, hookID.projectName, hookID.stackName, hookID.environmentName, hookID.webhookName)
 	if err != nil {
 		return nil, err
 	}
 	return webhook, nil
 }
 
 func splitWebhookID(id string) (*webhookID, error) {
-	// format: organization/project/stack/webhookName (stack webhook) or organization/webhookName (org webhook)
+	// format:
+	// organization/project/stack/webhookName (stack webhook)
```

Comment:
should we be consistent here and have the identifier always prefixed onto ID (e.g. stack vs env vs org) instead of inferring

Reply:
Unfortunately we have resources already created with those id's, so any format change would be breaking. I wish it could be consistent as well :c